test/server/certificate.pem

```diff
@@ -0,0 +1,16 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
these should probably live under `config`, not `test`. ie `config/server/...`

Reply:
True!